systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaBridgeUtils.java

```diff
@@ -64,11 +68,40 @@ public static void checkSendResponse(JsonObject response, int messageCount) {
         }
     }
 
-    public static void waitUntilKafkaBridgeStatus(String clusterName, String state) {
+    /**
+     * Wait until KafkaBridge is in desired state
+     * @param clusterName name of KafkaBridge cluster
+     * @param state desired state
+     */
+    public static void waitForKafkaBridgeStatus(String clusterName, String state) {
         LOGGER.info("Waiting till Kafka Bridge CR will be in state: {}", state);
         TestUtils.waitFor("Waiting for Kafka resource status is: " + state, Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
-            () -> KafkaBridgeResource.kafkaBridgeClient().inNamespace(kubeClient().getNamespace()).withName(clusterName).get().getStatus().getConditions().get(0).getType().equals(state)
+            () -> kafkaBridgeClient().inNamespace(namespace).withName(clusterName).get().getStatus().getConditions().get(0).getType().equals(state),
+            () -> StUtils.logCurrentStatus(kafkaBridgeClient().inNamespace(namespace).withName(clusterName).get())
         );
         LOGGER.info("Kafka Bridge CR is in state: {}", state);
     }
+
+    public static void waitForKafkaBridgeIsReady(String clusterName) {
+        waitForKafkaBridgeStatus(clusterName, "Ready");
+    }
+
+    public static void waitForKafkaBridgeIsNotReady(String clusterName) {
+        waitForKafkaBridgeStatus(clusterName, "NotReady");
+    }
+
+    /**
+     * Wait until KafkaBridge and its pods will be in Ready state
+     * @param clusterName name of KafkaBridge cluster
+     * @param expectPods number of expected pods to be ready
+     */
+    public static void waitForKafkaBridgeIsReady(String clusterName, int expectPods) {
```

Comment:
```suggestion
    public static void waitForKafkaBridgePodIsReady(String clusterName, int expectPods) {
```
This is maybe a thing that we can take into consideration. I think this could be generalized on the more abstract way. For instance, create a generic method in the `PodUtils`? What do you think @Frawless  ?

Reply:
The name is fine from my POV. Anyway do we need to wait here for pods after the resource is ready? AFAIU the resource shouldn't be ready when pods are not ready. And how do you want to generalize it? IT checks resource status and then use `PodUtils` to wait for pods

Reply:
@im-konge  @see-quick ^

Reply:
@Frawless I took look at it and the reason why I add waiting for pods was that `DeploymentUtils` have it same way. So there are two options from my POV:

1) remove it from `DeploymentUtils` as it is not necessary
2) we will let it on each `utils` to be sure it is really ready
![image](https://user-images.githubusercontent.com/53821852/79573478-03423180-80bf-11ea-9682-993a50fe024f.png)

Reply:
Well...the question is - do we want to check deployment status or just check Ready status of the bridge CR? In case of pure dpeloyment (clients for example) I would keep it there. My questions was mostly connected with CRs